docs/DesignDocs/LoggingFormat.md

```diff
@@ -0,0 +1,79 @@
+Logging Format
+=====
+
+When using the OE SDK and in trying to make use of the logs (e.g. via uploading them to Azure Log Analytics) it is nice to be able to parse and display all of the available logs including the ones generated by OE.
+
+Motivation
+----------
+
+Users should be able to specify and control their own logging format so that they can further parse the log files as they see fit.
+
+User Experience
+---------------
+
+- The user can set the `OE_LOG_DEVICE` environment variable to specify the log file location
+
+- The user can set the `OE_LOG_FORMAT` environment variable to specify the log format. The user will be given the below information:
+
+        - time in string format
+        - microseconds
+        - is enclave (E|H)
+        - log level
+        - thread id
+        - log message
+        - file log initiated from
+        - function log initiated from
+        - file line number log initiated from
+
+        so for example a log format could be set like this:
+
+        `OE_LOG_FORMAT=$'{\"e_ts\":\"%s.%06ldZ\",\"level\":\"(%s)%s\",\"tid\":\"tid(0x%lx)\",\"msg\":\"%s\",\"file\":\"%s\",\"func\":\"%s\",\"number":\"%s\"}\n'`
+
+        and end up looking like this:
+
+        `
+        {
+                "e_ts": "2019-09-24T10:09:27.832475Z",
+                "file": "switchless/host/host.c",
+                "func": "main",
+                "level": "(H)ERROR",
+                "msg": "Hello 'world'!",
+                "number": "79",
+                "tid": "tid(0x7fc2698a6d40)"
+        }
+        `
+
+- The user can set the `OE_LOG_ESCAPE` environment variable and if it is set then the log message will be escaped in order to be compatible with the JSON standard.
+
+Specification
+-------------
+
+Users can specify their own format when logging to a user specified file via the existing `OE_LOG_DEVICE` environment variable.
+If the var is not specified then logging goes on as usual.
```

Comment:
Not clear what `as usual` means.

Reply:
Meaning same behaviour as before this PR. Removed in order to avoid confusion.